app/models/katello/system.rb

```diff
@@ -63,6 +63,11 @@ class System < Katello::Model
 
     belongs_to :content_view, :inverse_of => :systems
 
+    has_one :capsule,
+            :class_name => "::SmartProxy",
+            :inverse_of => :system,
```

Comment:
Actually for now i would just refer to system everywhere, as that is what you've put in the migration